{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/debian-osv/dla-osv/DLA-3106-1.json"
      },
      "package": {
        "ecosystem": "Debian:10",
        "name": "python-oslo.utils",
        "purl": "pkg:deb/debian/python-oslo.utils?arch=source"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.36.5-0+deb10u2"
            }
          ],
          "type": "ECOSYSTEM"
        }
      ],
      "versions": [
        "3.36.4-2",
        "3.36.5-0+deb10u1"
      ]
    }
  ],
  "id": "DLA-3106-1",
  "modified": "2025-05-26T07:22:44.390901Z",
  "published": "2022-09-13T00:00:00Z",
  "schema_version": "1.7.3",
  "summary": "python-oslo.utils - security update",
  "upstream": [
    "CVE-2022-0718"
  ]
}